extension/src/support.c

```diff
@@ -16,6 +16,21 @@ PG_MODULE_MAGIC;
 #endif
 
 PG_FUNCTION_INFO_V1(make_call_subquery_support);
+PG_FUNCTION_INFO_V1(gapfill_delta_transition);
```

Comment:
This stuff is in the wrong .c file. It has nothing to do with support.

Reply:
I'd try to fix this not to need this too...

Reply:
hmm, I'm a bit worried that DLL export won't be set correctly if I move it...

Reply:
also, I don't think I can generate the function right now, `concat_idents!` is unstable